data/repos.yml

```diff
@@ -1293,8 +1293,11 @@
 
 - repo_name: smart-answers
   type: Frontend apps
-  team: "#govuk-publishing-on-platform-content-tech"
+  team: "#user-experience-measurement-govuk-robot-invasion"
+  shortname: smartanswers
   production_hosted_on: eks
+  argo_cd_apps:
```

Comment:
This was legacy config, does it need to go back in?

Reply:
No, but given that it isn't causing any adverse effects, it'll probably be easier to keep it as a part of this commit so that we can revert it again when the ownership stuff is settled.